src/components/icons/LinkOut.tsx

```diff
@@ -1,20 +1,14 @@
-                fill="#D3D3D3"
-            />
-        </svg>
-    );
-};
+import { faUpRightFromSquare } from "@fortawesome/free-solid-svg-icons/index";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
+/***
+ * NOTE: this currently gives a warning in the console:
+ * "Warning: FontAwesomeIcon: Support for defaultProps will be removed from function components
+ * in a future major release. Use JavaScript default parameters instead."
+ * I didn't write an issue because I can't reproduce it in codepen:
+ * https://codepen.io/meganrm/pen/BagJvXZ
+ */
+const LinkOut = () => (
+    <FontAwesomeIcon style={{ marginLeft: 6 }} icon={faUpRightFromSquare} />
```

Comment:
Consider adding some visually hidden text that says "(opens in new tab)" as part of this icon? Or add alt text to it that describes that.

https://www.w3.org/TR/WCAG20-TECHS/H83.html
https://webaim.org/techniques/css/invisiblecontent/